rest/src/main/kotlin/builder/interaction/PublicInteractionBuilder.kt

```diff
@@ -36,6 +38,12 @@ class PublicInteractionResponseCreateBuilder :
     private var _tts: OptionalBoolean = OptionalBoolean.Missing
     var tts: Boolean? by ::_tts.delegate()
 
+    @KordPreview
+    private var _components: Optional<MutableList<DiscordComponent>> = Optional.Missing()
+
+    @KordPreview
+    var components: MutableList<DiscordComponent>? by ::_components.delegate()
```

Comment:
* `var: MutableList` should be replaced with `val: MutableList`

* `Optional<List>` should be avoided because `Value(emptyList())` doesn't make sense for an optional field. Remove the optional and assume that empty means missing.